sklearn/tree/_export.py

```diff
@@ -283,7 +283,8 @@ def node_to_str(self, tree, node_id, criterion):
         if tree.children_left[node_id] != _tree.TREE_LEAF:
             # Always write node decision criteria, except for leaves
             if self.feature_names is not None:
-                feature = self.feature_names[tree.feature[node_id]]
+                feature = str(self.feature_names[tree.feature[node_id]])
```

Comment:
Why is this `str()` call required? Can `feature_names` be integers in this context?

Reply:
I _think_ they should always be strings, and if they are not, they'll probably cause a lot of other problems. I'll remove the call.